app/models/form/question.rb

```diff
@@ -195,7 +197,7 @@ def value_is_refused?(value)
   end
 
   def display_label
-    check_answer_label || header || id.humanize
+    error_label || check_answer_label || header || id.humanize
```

Comment:
Is display label only for errors? If so, maybe worth renaming it something more specific?
Also, would it be possible to avoid adding error_label attribute in general by updating check_answers_label with correct display_label?

Reply:
Yes to the first bit - I'll look at renaming
No to the second bit, this design requires it as we have a different label for the error message (address line 1) to the cya label (address lines 1 and 2)